### PR TITLE
[Xamarin.Android.Build.Tasks] include openjdk link in error messages

### PR DESCRIPTION
**Pull request by jonathanpeppers**
`XA0031` doesn't include a download link. Let's make the error message more helpful.

### CODE REVIEW

File: src/Xamarin.Android.Build.Tasks/Properties/Resources.resx [hunk 1]
```diff
@@ -178,13 +178,13 @@ In this message, the phrase "should not be reached" means that this error messag
...
   <data name="XA0031" xml:space="preserve">
-    <value>Java SDK {0} or above is required when using {1}.  Note: the Android Designer is incompatible with Java SDK 11.0: https://aka.ms/vs2019-and-jdk-11</value>
+    <value>Java SDK {0} or above is required when using {1}. Download the latest JDK at https://www.microsoft.com/openjdk  Note: the Android Designer is incompatible with Java SDK 11.0: https://aka.ms/vs2019-and-jdk-11</value>
     <comment>
 {0} - The Java SDK version number
 {1} - The feature being used: `$(TargetFrameworkVersion) v10.0`</comment>
   </data>
   <data name="XA0031_NET" xml:space="preserve">
-    <value>Java SDK {0} or above is required when using .NET 6 or higher.</value>
+    <value>Java SDK {0} or above is required when using .NET 6 or higher. Download the latest JDK at: https://www.microsoft.com/openjdk</value>
```
Comment (jonathanpeppers) [inline src/Xamarin.Android.Build.Tasks/Properties/Resources.resx:187]: Let me know if there is a better way to phrase these.

Comment (mcumming) [reply src/Xamarin.Android.Build.Tasks/Properties/Resources.resx:187]: Should we be using the https://aka.ms/msopenjdk link instead of the direct url?

Comment (jonathanpeppers) [reply src/Xamarin.Android.Build.Tasks/Properties/Resources.resx]: Thanks, update the URL.

**Review by dellis1972 — APPROVED** [review]

**Comment by jonpryor** [comment]
Related:

  * d43128c2e58cc9f350bb2c0c19c974e89cd79269
  * https://user-images.githubusercontent.com/155958/55502164-f5d41980-5619-11e9-8df0-200c874e180a.png
  * https://user-images.githubusercontent.com/155958/55503495-c07cfb00-561c-11e9-84ac-fc1af2ef5917.png
  * https://github.com/xamarin/xamarin-android/pull/5162#discussion_r499000175
  * https://github.com/xamarin/xamarin-android/pull/4614#discussion_r471700258

I think we should make these updated messages *multi-line* affairs.  Otherwise they will be so long, very few will actually read the URLs within it.

**Pull request merged**